PE/__init__.py

```diff
@@ -10,6 +10,9 @@
 
 from . import ordlookup
 
+PE32_MAGIC = 0x10b
+PE32PLUS_MAGIC = 0x20b
+
 IMAGE_FILE_RELOCS_STRIPPED = 0x0001
 IMAGE_FILE_EXECUTABLE_IMAGE = 0x0002
 IMAGE_FILE_LINE_NUMS_STRIPPED = 0x0004
```

Comment:
not necessary for this PR, but it smells like half this file should be incorporated into a PE.consts module

Reply:
Definitely should, but yea, best saved for a PR that's intended for more than just a quick bugfix.